### ISSUE TITLE
GDT cannot be installed for Eclipse 4.27 (2023-03)

### ISSUE THREAD

**Issue by richip** [groovy/groovy-eclipse#1456]
As stated in the title, I cannot install groovy-eclipse on fresh install of Eclipse 4.27 (2023-03) or any of the previous 2 versions that I've tried either. I've tried installing via Marketplace and using the Update site https://groovy.jfrog.io/artifactory/plugins-snapshot/e4.27 ...
The error I get is:

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Eclipse Groovy Development Tools 4.9.0.v202302150038-e2303 (org.codehaus.groovy.eclipse.feature.feature.group 4.9.0.v202302150038-e2303)
  Missing requirement: Eclipse JDT Core patch for Groovy 4.9.0.v202302150038-e2303 (org.codehaus.groovy.jdt.patch.feature.group 4.9.0.v202302150038-e2303) requires 'org.eclipse.equinox.p2.iu; org.eclipse.jdt.feature.group [3.19.0.v20230104-1800,3.19.0.v20230104-1800]' but it could not be found
  Cannot satisfy dependency:
    From: Groovy-Eclipse Compilerless feature 4.9.0.v202302150038-e2303 (org.codehaus.groovy.compilerless.feature.feature.group 4.9.0.v202302150038-e2303)
    To: org.eclipse.equinox.p2.iu; org.codehaus.groovy.headless.feature.feature.group [4.9.0.v202302150038-e2303,4.9.0.v202302150038-e2303]
  Cannot satisfy dependency:
    From: Eclipse Groovy Development Tools 4.9.0.v202302150038-e2303 (org.codehaus.groovy.eclipse.feature.feature.group 4.9.0.v202302150038-e2303)
    To: org.eclipse.equinox.p2.iu; org.codehaus.groovy.compilerless.feature.feature.group [4.9.0.v202302150038-e2303,4.9.0.v202302150038-e2303]
  Cannot satisfy dependency:
    From: Groovy-Eclipse Headless feature 4.9.0.v202302150038-e2303 (org.codehaus.groovy.headless.feature.feature.group 4.9.0.v202302150038-e2303)
    To: org.eclipse.equinox.p2.iu; org.codehaus.groovy.jdt.patch.feature.group [4.9.0.v202302150038-e2303,4.9.0.v202302150038-e2303]
```

I just want to be able to install and use it as I'd like to learn Groovy programming, so feel free to suggest any version of Eclipse and groovy-eclipse that I can install.

**Comment by eric-milles** [comment]
4.27 is in the milestone phase, so it does not make for a good learning environment.  I'll update to M3 (just dropped this week) in short order.

4.26 cannot be installed from a packaged release.  Unfortunately, I cannot fix this: #1425 

4.25 is quite stable and has snapshots available so you can get latest bug fixes.  **I recommend using 4.25 (2022-09).**

**Comment by richip** [comment]
> 4.25 is quite stable and has snapshots available so you can get latest bug fixes. **I recommend using 4.25 (2022-09).**

Thank you very much. I can confirm that a fresh install of 2022-09 works for installing GDT.
